drivers/MbedCRC.h

```diff
@@ -193,7 +193,7 @@ class MbedCRC {
      *  @note: CRC as output in compute_partial is not final CRC value, call `compute_partial_stop`
      *         to get final correct CRC value.
      */
-    int32_t compute_partial(void *buffer, crc_data_size_t size, uint32_t *crc)
+    int32_t compute_partial(const void *buffer, crc_data_size_t size, uint32_t *crc)
```

Comment:
The constness gets unnecessarily thrown away by the C-style cast in the `hal_crc_compute_partial` call below. Can tighten that up by making it

                hal_crc_compute_partial(static_cast<const uint8_t *>(buffer), size);

as `static_cast` won't let you drop the `const`.